NightscoutServiceKit/RemoteCommands/RemoteCommandSourceV1.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  RemoteCommandSourceV1.swift
+//  NightscoutServiceKit
+//
+//  Created by Bill Gestrich on 2/25/23.
+//  Copyright © 2023 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import LoopKit
+import NightscoutUploadKit
+
+class RemoteCommandSourceV1: RemoteCommandSource {
+    
+    private let otpManager: OTPManager
+    
+    init(otpManager: OTPManager) {
+        self.otpManager = otpManager
+    }
+    
+    
+    //MARK: RemoteCommandSource
+    
```

Comment:
I need a way to distinguish Remote 1. 0 and Remote 2.0 push notifications. I'm planning to add a "version" string to the push notification in Remote 2.0.